content/en/docs/concepts/overview/components.md

```diff
@@ -23,13 +23,13 @@ Here's the diagram of a Kubernetes cluster with all the components tied together
 {{% /capture %}}
 
 {{% capture body %}}
-## Master Components
+## Control Plane Components
 
-Master components provide the cluster's control plane. Master components make global decisions about the
+Control Plane components provide the cluster's control plane. Control Plane components make global decisions about the
```

Comment:
“Control Plane components provide the cluster's control plane”? @Fale, the change makes this sentence tautological IMO.

Maybe use `{{< glossary_definition term_id="control-plane" length="all" >}}` here? You might want / need to reword the glossary definition slightly to make it work.